### PR TITLE
refine act conv2d pool2d trt converter log

### PR DESCRIPTION
**Pull request by NHZlX**
Error message refine in conv2d, pool2d, fc, act trt converter .

### CODE REVIEW

File: paddle/fluid/inference/tensorrt/convert/conv2d_op.cc [hunk 1]
```diff
@@ -133,10 +148,11 @@ class Deconv2dOpConverter : public OpConverter {
           return layer;
         },
         [](nvinfer1::IDeconvolutionLayer* layer, nvinfer1::DimsHW& dilations) {
-          PADDLE_ENFORCE(
-              dilations.d[0] == 1 && dilations.d[1] == 1,
-              "Dilations must be (1, 1) for tensorRT, but given (%d, %d)",
-              dilations.d[0], dilations.d[1]);
+          PADDLE_ENFORCE_EQ(
```
Comment (Superjomn) [inline paddle/fluid/inference/tensorrt/convert/conv2d_op.cc]: PADDLE_ENFORCE ? 

Split to two enforces to helper tell whether dilations[0] or dilations[1] is wrong.

File: paddle/fluid/inference/tensorrt/convert/pool2d_op.cc [hunk 2]
```diff
@@ -56,15 +56,19 @@ class Pool2dOpConverter : public OpConverter {
...
         << "convert a fluid pool2d op to tensorrt pool2d layer without bias";
     framework::OpDesc op_desc(op, nullptr);
-    // Declare inputs
-    PADDLE_ENFORCE_EQ(op_desc.Input("X").size(), 1);
-    PADDLE_ENFORCE_EQ(op_desc.Output("Out").size(), 1);
+    PADDLE_ENFORCE_EQ(op_desc.Input("X").size(), 1,
+                      platform::errors::InvalidArgument(
+                          "TRT Pool2d expect 1 input, but got %d input.",
+                          op_desc.Input("X").size()));
+    PADDLE_ENFORCE_EQ(op_desc.Output("Out").size(), 1,
```
Comment (Superjomn) [inline paddle/fluid/inference/tensorrt/convert/pool2d_op.cc]: 1UL

**Review by Shixiaowei02 — DISMISSED** [review]

File: paddle/fluid/inference/tensorrt/convert/conv2d_op.cc (hunk 1 above)
Comment (NHZlX) [reply paddle/fluid/inference/tensorrt/convert/conv2d_op.cc]: PADDLE_ENFORCE 现在是禁用的。 已经修改了下

File: paddle/fluid/inference/tensorrt/convert/pool2d_op.cc (hunk 2 above)
Comment (NHZlX) [reply paddle/fluid/inference/tensorrt/convert/pool2d_op.cc]: done

**Review by Shixiaowei02 — APPROVED** [review]

**Pull request merged**